plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -227,8 +235,8 @@ def updateFirmware(self, file_name):
     def firmwareUpdateFinished(self):
         return self._firmware_update_finished
 
-    def resetFirmwareUpdateFinished(self):
-        self._firmware_update_finished = False
+    def resetFirmwareUpdate(self, update_has_finished = False):
```

Comment:
No problem. To be honest I just saw this code while working on _updateFirmware and decided to make it more common.